src/runner.js

```diff
@@ -132,11 +132,14 @@ exports.run = function(config) {
     // establish ngrok tunnel and resolve ngrok url
     .then(function(proxyHost) {
       if (config.tunnel) {
-        console.log('Connecting tunnel');
-        return Tunnel.connect(proxyHost || config.tunnel.host, config.tunnel.token);
-      } else {
-        return Promise.resolve();
+        console.log('Connecting ngrok tunnel');
```

Comment:
Please change this back to just `Connecting tunnel`. Users may not know what "ngrok" is and get confused

Reply:
changed